### DISCUSSION TITLE
overlay editor field mouse scrolling

### DISCUSSION THREAD

**Discussion by ssagerian** [AllskyTeam/allsky-modules#66] [General]
I noticed that when my mouse is hovering inside the overlay editor image field, that scrolling doesn't work. It works fine when the mouse is outside the perimeter of the image. This is using V20230501_03

**Comment by Alex-developer** [comment]
Thanks for the report. This was initially by design but I cant recall why !!!!

I have logged it as an issue and will look at it under that (https://github.com/AllskyTeam/allsky-modules/issues/69)
